WcaOnRails/app/models/competition.rb

```diff
@@ -54,6 +55,13 @@ def must_have_at_least_one_delegate
     end
   end
 
+  validate :delegates_must_be_delegates, on: [:update]
```

Comment:
Why is this only on update? That gets me thinking... what if we're cloning a competition that has someone who is no longer a delegate?